haproxy/datadog_checks/haproxy/version_utils.py

```diff
@@ -0,0 +1,21 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import re
+
+VERSION_PATTERN = re.compile(r"(?:HAProxy|hapee-lb) version ([^,]+)")
+
+
+def get_version_from_http(raw_version):
```

Comment:
(Nit) I find the naming a bit confusing here — initially I thought this would make an HTTP request and parse the result, but this only does the parsing bit.

```suggestion
def parse_version_from_http_line(raw_version):
```